docs/examples/logging-insights/README.md

```diff
@@ -246,22 +246,37 @@ We'll deploy first to Docker to test.
 
 
 ```python
-!ls artifacts/conda.yaml
+%%writefile artifacts/conda.yaml
+name: tempo-insights
+channels:
+  - defaults
+dependencies:
+  - pip=21.0.1
+  - python=3.7.9
+  - pip:
+    - mlops-tempo @ file:///home/alejandro/Programming/kubernetes/seldon/tempo
```

Comment:
Should this be reverted back to the regular file?

Reply:
Good shout, I think this should be remoed actually as its created by the notebook

Reply:
Done